### PR TITLE
2 cars added

### PR DESCRIPTION
**Pull request by SRINIVAS383**

### CODE REVIEW

File: showroom [hunk 1]
```diff
@@ -1,3 +1,5 @@
 audi 
 benz
 rolls roice
+ferrari
+range rover
```
Comment (vasu-devop) [inline showroom:5]: i need more brand new cars added in the list asap

**Review by vasu-devop — COMMENTED** [review]
asaap

File: showroom (hunk 1 above)
Comment (SRINIVAS383) [reply showroom:5]: okay easy

File: showroom [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+lambogini
+swift
+innova
+tata
+f1 racer
```
Comment (vasu-devop) [inline showroom:10]: done

**Review by vasu-devop — COMMENTED** [review]
done

**Pull request merged**